packages/react-hotkeys-hook/src/lib/parseHotkeys.ts

```diff
@@ -33,11 +33,22 @@ export function parseKeysHookInput(keys: string, delimiter = ','): string[] {
   return keys.toLowerCase().split(delimiter)
 }
 
-export function parseHotkey(hotkey: string, splitKey = '+', useKey = false, description?: string): Hotkey {
-  const keys = hotkey
-    .toLocaleLowerCase()
-    .split(splitKey)
-    .map((k) => mapCode(k))
+export function parseHotkey(hotkey: string, splitKey = '+', sequenceSplitKey = '>', useKey = false, description?: string): Hotkey {
+  let keys: string[] = []
+  let isSequence = false
+
+  if (hotkey.includes(sequenceSplitKey)) {
+    isSequence = true
+    keys = hotkey
+      .toLocaleLowerCase()
+      .split(sequenceSplitKey)
+      .map((k) => mapCode(k))
+  } else {
+    keys = hotkey
+      .toLocaleLowerCase()
+      .split(splitKey)
+      .map((k) => mapCode(k))
```

Comment:
This can be simplified since only the split call is different

Reply:
I can change this to assign a var for the splitKey and isSequence bool based on if the hotkey includes sequence split key or not, but I think this version is easier to read.

Reply:
fair enough